src/Microsoft.PowerShell.Commands.Utility/commands/utility/Send-MailMessage.cs

```diff
@@ -206,6 +208,20 @@ public String Subject
         }
         private String _subject;
 
+        /// <summary>
+        /// Specifies that the mail message is sent without a subject.
+        /// </summary>
+        [Parameter(ValueFromPipelineByPropertyName = true, ParameterSetName = "NoSubject")]
+        public SwitchParameter NoSubject
+        {
+            get { return _nosubject; }
+            set
+            {
+                _nosubject = value;
+            }
+        }
+        private SwitchParameter _nosubject;
```

Comment:
@ThreeFive-O If you plan to continue please use auto property and remove the private field.